openeogeotrellis/load_stac.py

```diff
@@ -211,6 +214,13 @@ def load_stac(
 
                 item_collection = ItemCollection.from_stac_catalog(catalog, spatiotemporal_extent=spatiotemporal_extent)
 
+        # Deduplicate items
+        # TODO: smarter and more fine-grained deduplication behavior?
+        #       - enable by default or only do it on STAC API usage?
+        #       - allow custom deduplicators (e.g. based on layer catalog info about openeo collections)
+        if feature_flags.get("deduplicate-items", False):
```

Comment:
rename to "deduplicate_items" for better consistency with other feature flags and parameters